### PR TITLE
Admin horizontal scroll on narrower screens

### LINKED ISSUE #3260: Admin scrolls horizontally in a weird way on narrower screens
### Share your bug report, feature request, or comment.

I don't have the largest laptop screen in the world, and when I open a sidebar it makes the admin scroll in a very odd way. It should dynamically adjust spacing to allow more to fit instead of just cutting it off.

image

### PR DESCRIPTION
**Pull request by anniepauline**
**What type of PR is this?**
> /kind design

**Any specific area of the project related to this PR?**
> /area frontend

**Which issue(s) this PR fixes**:
This PR fixes the responsive design on the admin page when the screen gets narrower

Usage: Fixes [/issues/3260]

### CODE REVIEW

**Comment by gabek** [comment]
Hi there. It looks like you accidentally committed more files than you intended to. It says 68 files were changed, but I'm guessing you only changed one or two files.

**Comment by anniepauline** [comment]
> Hi there. It looks like you accidentally committed more files than you intended to. It says 68 files were changed, but I'm guessing

Hello @gabek !! Yes, I'll make the necessary changes

**Comment by anniepauline** [comment]
@gabek Hello!!
I made the necessary changes. Please check it out!

**Comment by gabek** [comment]
Awesome, thank you! Just as a heads-up, I probably won't have a chance to test PRs for a week or so, but I just wanted you to know, so you don't think that I'm ignoring it.

**Comment by gabek** [comment]
Hi @anniepauline, it looks like there's a linter warning. Could you look at that?

**Comment by anniepauline** [comment]
> Hi @anniepauline, it looks like there's a linter warning. Could you look at that?
I am not sure how to fix linting errors. Can you please guide me through this?
is this the command I must use ? `npm init @eslint/config`

**Comment by gabek** [comment]
If you use `eslint` in your editor, it should hopefully point out the issue. I'm guessing it points to a sytax error somewhere, but I'm not sure exactly what this error means specifically. Hopefully an editor integration will point something useful out.

Here's the full error:

```
/home/runner/work/owncast/owncast/web/public/styles/admin/config-public-details.css
  1:0  error  Parsing error: Declaration or statement expected
```

**Comment by anniepauline** [comment]
> If you use `eslint` in your editor, it should hopefully point out the issue. I'm guessing it points to a sytax error somewhere, but I'm not sure exactly what this error means specifically. Hopefully an editor integration will point something useful out.
> 
> Here's the full error:
> 
> ```
> /home/runner/work/owncast/owncast/web/public/styles/admin/config-public-details.css
>   1:0  error  Parsing error: Declaration or statement expected
> ```

Hey @gabek I've made a new commit!! Please check it out

**Comment by gabek** [comment]
Is it because you removed the newline? I'm only guessing. Tonight I'll open up your PR and figure out what the actual issue is.

image

**Comment by anniepauline** [comment]
> Is it because you removed the newline? I'm only guessing. Tonight I'll open up your PR and figure out what the actual issue is.
> 
> image

Maybe, I tried removing and adding the new line, but the issue still persists

**Comment by gabek** [comment]
The good news is it had nothing to do with your change. The linter complained on the file even without it. The bad news is I have no idea what's going on. I'm guessing it's some bug with the linter. So I just had the linter ignore it.

**Comment by gabek** [comment]
Small tweak! It looks like on my particular browser window there's still room for the content on the right, but it has jumped to the single column layout. Could it be changed a bit to not be so sensitive before it wraps around?

Sorry for the delays and the dumb linter.

image

**Comment by anniepauline** [comment]
@gabek no worries!! I'll get working on it!

File: web/public/styles/admin/config-public-details.css [hunk 1]
```diff
@@ -19,55 +23,75 @@
     width: 100%;
     max-width: none;
   }
+
+  /* .social-items-container {
+    overflow: hidden;
+  } */
+
+
```
Comment (gabek) [inline web/public/styles/admin/config-public-details.css]:
```suggestion
```

**Review by gabek — APPROVED** [review]

**Comment by gabek** [comment]
Works great! Thanks so much for continuing to work on this and make changes. I hope you want to continue to help with Owncast over time. Thanks again!

**Pull request merged**